src/install/memcached/linux/install-apt.mdx

```diff
@@ -0,0 +1,14 @@
+---
+componentType: default
+headingText: Download using APT
+---
+
+1. From the command line, run these commands:
+
+    ```shell
+    sudo apt-get update
+    ```
+
+    ```shell
+    sudo apt-get install nri-memcached
+```
```

Comment:
indent the last set of backticks here. it's adding a secondary / random (i think) code block on the preview that reads:

`export const _frontmatter = {"componentType":"default","headingText":"Download using APT"}`